pyramid/decorator.py

```diff
@@ -19,17 +19,22 @@ def jammy(self):
     And usage of Foo:
 
     >>> f = Foo()
-    >>> v = f.jammy
+    >>> print(f.jammy)
```

Comment:
This change doesn't feel right:  having `jammy` do the print means we should be doing something like assigning `v = f.jammy`, and then maybe `print(v)`.